src/apps/frontend/pages/authentication/login/login-form.tsx

```diff
@@ -0,0 +1,85 @@
+import { Block } from 'baseui/block';
+import { KIND } from 'baseui/button';
+import { FormControl } from 'baseui/form-control';
+import { Input } from 'baseui/input';
+import { HeadingMedium } from 'baseui/typography';
+import React from 'react';
+
+import {
+  Button,
+} from '../../../components';
+
+import useLoginForm from './login-form.hook';
+
+interface LoginFormProps {
+  onSuccess: () => void;
+  onError: (error: string) => void;
```

Comment:
Can we pass error object instead of string?

Reply:
Done